### ISSUE TITLE
grml.github.io needs some love

### ISSUE THREAD

**Issue by mika** [grml/grml.github.com#3]
https://grml.github.io/ needs some love and updates, it's >7 years old and some of it information is no longer accurate.

**Comment by zeha** [comment]
@mika maybe sth to discuss this week?

**Comment by mika** [comment]
Related issue: https://github.com/grml/grml.github.com/issues/5

**Comment by jkirk** [comment]
We decided to migrate grml.github.io to https://grml.org/contribute, see: grml/grml.org#70.

Should we handle everything in there and close this issue?

**Comment by mika** [comment]
Yeah, let's continue in https://github.com/grml/grml.org/issues/70 but let's keep this open until we took care of this, so we properly discontinue this repsoitory? :)
